### PR TITLE
chekups

### PR DESCRIPTION
**Pull request by Alarlar**

### CODE REVIEW

File: CSS/index.css [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+  margin-top: 50px;  
 }
 
+/*
 section {
   padding: 1em 2em;
   margin-bottom: 2em;
 }
-
+*/
```
Comment (brandaogabriel7) [inline CSS/index.css]: Make sure you don't need these styles anymore and remove them to make your code cleaner.

File: CSS/index.css [hunk 2]
```diff
@@ -37,32 +39,33 @@ nav a {
...
+/*
 .connect-list {
   display: flex;
   justify-content: space-between; 
   align-items: center; 
 }
-
+*/
```
Comment (brandaogabriel7) [inline CSS/index.css]: Remove this commented out code.

File: CSS/index.css [hunk 3]
```diff
@@ -37,32 +39,33 @@ nav a {
...
+/*
 .experience-item {
   background-color: #fff;
   padding: 15px;
   border-radius: 5px;
   flex: 1 0 30%; 
 }
-
+*/
+*/
```
Comment (brandaogabriel7) [inline CSS/index.css]: Remove this commented out code.

File: index.html
Comment (brandaogabriel7) [inline index.html:1]: The indentation for this file is inconsistent. Try formatting it automatically using VSCode shortcut (Alt + Shift + F).

**Review by brandaogabriel7 — COMMENTED** [review]
## Lesson 10 - CSS Basics
Nice Job! You were able to correctly import your CSS stylesheet and fix most of the issues from before.

Still, you didn't fulfill all the requirements for this lesson. The main thing I want to make sure you understand is how to select elements relative to their containers/parents. You can look at the checklist below to see this and other requirements that are missing.

Also, one thing to be mindful about is that leaving commented out code can pollute your files, leave stuff will never be used and makes your files harder to read and understand. So, if you're not using it, just remove it.

### Create and Load Stylesheet
- [x] Create a folder called css at the same level as your README.md and index.html files
- [x] Inside that folder, create a Cascading Style Sheets (CSS) file called index.css
- [x] Open your index.html file
- [x] Before the closing </head> tag, insert a <link> element with a rel attribute of "stylesheet" and an href attribute that specifies the relative path to your CSS file (i.e. css/index.css)
### Write CSS
- [x] Change the background color of the page body
- [ ] Change the default text color
  > You only changed the text color for links and nav items. To change the default text color for the whole page, you can set it in the body styles.
- [x] Customize the font family
  - [ ] STRETCH GOAL (optional): load in a font family from [Google Fonts](https://fonts.google.com/)
- [x] Add spacing (padding/margin) between sections
- [x] Change the alignment of the content of one of your sections
- [ ] Change the font size, weight, and color of headings
> You didn't change any of these for your headings.
- [x] Transform the style of your Name at the top of the page
  - [ ] STRETCH GOAL (optional): add a picture of yourself to HTML and/or CSS (remember to include accessibility aspects if you add any images!)
- [ ] Transform the "Experience" list items into styled blocks
> You didn't style your list items, you styled the links for the whole page and that applied to the links in those list items. What you need to do here is change the styles to list items (`li` elements) inside your Experience section.
> 
> I saw that you added the class `experience-item` to each of them, but a simpler approach would be to select them relatively to their parent elements (the `#Experience` section). That way you could add many other items without having to repeat this class for all of them.
- [ ] Transform the style of the "Connect" links
  > These links are styled because of the styles added with the `a` selector. But I would like you to try and style only links inside the `Connect` section, not all links on the page.
  - [ ] STRETCH GOAL (optional): make or use social media icons to replace your link text with images

**Comment by brandaogabriel7** [comment]
## Lesson 11 - CSS Layout
You did a good job implementing the navigation!

The other goal besides the navigation was about using Flexbox to enchance your page layout. You did use Flexbox, but not in the way the requirements described. Take a look at the checklist below for notes on the unfulfilled requirements.

- [x] In your index.html file, add a navigational header to your webpage using the `<nav>` element.
- [ ] Add two sections to your index.html file:
    - [x] The first section will be "Skills". Use an h2 tag as you did with your About, Experience, and Connect sections, and be sure to include the id property in the element. You can leave the section empty for now. In later weeks we'll use JavaScript to insert a list of your skills.
    - [ ] The second section will be "Projects". Use an h2 tag as you did with your About, Experience, and Connect sections, and be sure to include the id property in the element. You can leave the section empty for now. In later weeks we'll use JavaScript to pull a list of your project repositories from your GitHub account using their API.
        > You forgot to add the Projects section.
- [x] The navigation menu should include a link to each of your sections (i.e. "About", "Experience", "Skills", "Projects", "Connect")
    - [x] STRETCH GOAL (Optional) Make the header sticky/fixed on the page using code in your index.css file.
- [ ] In your index.css file, change the layout of your "Experience" section using rows and columns so items are displayed in a better layout rather than a list Hint: use Flexbox - we suggest have job titles to the left, dates worked that job to the right and brief description of the job below the title.
> Your experience section content is not what's expected. Instead of the links you're using, you should describe your work experience with job title and date. Then you would use flexbox as described above.
- [ ] Also in your index.css file, update the "Connect" section to use Flexbox to improve the layout of your social media icons or links **NOTE:** This is the basic rubric, but we encourage you to think beyond this list and come up with your own ideas to make your webpage unique!
> You're not styling your `Connect` section. You do have some styles to the `.connect-list` but they're commented out.

**Comment by brandaogabriel7** [comment]
## Lesson 12 - Responsive Design
Excellent! You fulfilled all the requirements for this lesson. Looks like you understand what are media queries work and how to use them to change the way your website looks in each device.

- [x] Open your index.css file
- [x] Determine if you'd like to build your site mobile-first or desktop-first (meaning the code you have in there right now will be the default code for a phone or a desktop/laptop monitor)
- [x] Add comment notation to help keep yourself organized and make it easier to find your desktop/tablet/mobile style code sections
- [x] Based on if you decided to design mobile-first or desktop first, add your media queries for the other two device types. Example: if you're designing mobile-first, you'll want to add a media query that handles devices that are tablet sized, and a media query that handles anything larger than the largest tablet size.
- [x] Make a minimum of two changes to at least three elements for each of your media queries. Example: In your tablet media query, change the font size (1st change) and color (2nd change) of your headers (1st element), change the padding or margin (1st change) and background color (2nd change) of your unordered lists (2nd element), and change the font size (1st change) and font style (2nd change) of your name at the top of your page (3rd element)
    - [ ] STRETCH GOAL: Since we learned about grid layout this week, reformat one or both of your two flexbox sections (Experience or Connect) to be a grid layout instead to give yourself practice using grid.

**Comment by brandaogabriel7** [comment]
## Lesson 13 - The DOM API
This looks great, Dinar! Excellent job!

### Create a JavaScript file
- [x] Create a folder called `js` at the same level as your index.html, readme.md, and your css folder.
- [x] Inside the js folder, create a JavaScript file called `index.js`
- [x] Open your `index.html` file
- [x] Before the closing `</body>` tag, insert a `<script>` element with a `src` attribute that specifies the relative path to your JavaScript file (i.e. `js/index.js`)
- [x] Save and open the index.html in your browser so you can check your changes to your html page as you build the js code below. _If you're using VSCode or similar, you should be able to install/may already have an extension that allows you to have a "live" version of your index.html running in your browser that will automatically update when changes in your files are saved so you don't have to keep refreshing your file in your browser._

### Add a Footer Element
- [x] In your index.js file, using DOM manipulation, add a 'footer' child element to your index.html. Be careful on this step, as different DOM methods (append, appendChild, lastChild, etc.) have different placements.
- [x] Remember that you can store the elements in variables to reference the variables and call methods on them to more easily follow your code. (It's easier to read `body.innerHTML` than it is to read `document.getElementsByTagName("body").innerHTML`)

#### Insert Copyright Text in Footer
- [x] Open your `index.js` file
- [x] Create a variable named `today` and assign it a new date object
    - hint: `new Date()` constructor
- [x] Create a variable named `thisYear` and assign it the current year from your date object. Use a method to do this, don't hardcode "2024" because next year your website will have the wrong year!
    - hint: `getFullYear` method
- [x] Create a variable named `footer` and assign it the footer element by using "DOM Selection" to select the `<footer>` element from the DOM
    - hint: `querySelector` method or similar
- [x] Create a variable named `copyright` and use it to create a new paragraph (`p`) element
    - hint: `createElement` method
- [x] Set the inner HTML of your `copyright` element to display your name and the current year
    - hint: use the `copyright` variable and the `thisYear` variable from earlier
- [x] Append the `copyright` element to the footer using "DOM Manipulation"
    - hint: `appendChild` method or similar
    - [x] STRETCH GOAL: Use unicode to also include the copyright symbol ( © ) in your footer content
- [x] Save and refresh your browser
    - You should see the text "_Your Name_ 2024" at the bottom of the page

### Add to your Skills Section

- [x] Open your `index.html` file
- [x] Above the "Connect" section, add a new `<section>` element with an `id` attribute of value "skills"
- [x] Inside the new section, add a `<h2>` element that says "Skills"
- [x] After the `<h2>` element, add an empty unordered list (`<ul>`) element
- [x] Save and refresh your browser _(or just check your browser for changes if using live extension)_
    - You should see the new "Skills" heading

### Create List of Skills

- [x] Open your `index.js` file
- [x] List your technical skills by creating an Array of String values and store it in a variable named `skills`
    - Example: `["JavaScript", "HTML", "CSS", "Adobe Photoshop", "GitHub"]`
- [x] Create a variable named `skillsSection` and use "DOM Selection" to select the skills section by id
    - hint: `querySelector` or `getElementById` method
- [x] Create a variable named `skillsList` and use "DOM Selection" to query the `skillsSection` (instead of the entire `document`) to select the `<ul>` element you created earlier in this assignment
- [x] Create a `for` loop to iterate over your `skills` Array
- [x] Inside the loop, create a variable named `skill` to create a new list item (`li`) element
    - hint: `createElement` method
- [x] Still inside the loop, use the `skill` variable to set the inner text to the current Array element
    - hint: access the Array element using bracket notation
- [x]  The last thing we'll do inside the loop is append the `skill` element to the `skillsList` element
    - hint: `appendChild` method
- [x] Save and refresh your browser _(or just check your browser for changes if using live extension)_
    - You should see your list of skills beneath the "Skills" heading

**Review by brandaogabriel7 — CHANGES_REQUESTED** [review]
You're doing great, Dinar!
Both assignments 12 and 13 look great! But you still need to review a few things in lessons 10 and 11. I left a comment with the assignment checklists for each of them.

File: CSS/index.css (hunk 1 above)
Comment (Alarlar) [reply CSS/index.css]: Got it.

**Comment by brandaogabriel7** [comment]
## Lesson 10 - CSS Basics
Nice job! Now, the only issues left are:
1. Changing the default text color for the page.
2. Changing the styles for headings.

There are comments that explain both of those issues. Below is the updated checklist.

### Create and Load Stylesheet
- [x] Create a folder called css at the same level as your README.md and index.html files
- [x] Inside that folder, create a Cascading Style Sheets (CSS) file called index.css
- [x] Open your index.html file
- [x] Before the closing </head> tag, insert a <link> element with a rel attribute of "stylesheet" and an href attribute that specifies the relative path to your CSS file (i.e. css/index.css)
### Write CSS
- [x] Change the background color of the page body
- [ ] Change the default text color
  > You only changed the text color for links and nav items. To change the default text color for the whole page, you can set it in the body styles.
- [x] Customize the font family
  - [ ] STRETCH GOAL (optional): load in a font family from [Google Fonts](https://fonts.google.com/)
- [x] Add spacing (padding/margin) between sections
- [x] Change the alignment of the content of one of your sections
- [ ] Change the font size, weight, and color of headings
> You did change the heading styles but you used the same styles for all of them and made your name disappear. Take a look at the comment I left about that.
- [x] Transform the style of your Name at the top of the page
  - [ ] STRETCH GOAL (optional): add a picture of yourself to HTML and/or CSS (remember to include accessibility aspects if you add any images!)
- [x] Transform the "Experience" list items into styled blocksto their parent elements (the `#Experience` section). That way you could add many other items without having to repeat this class for all of them.
- [x] Transform the style of the "Connect" links
  - [ ] STRETCH GOAL (optional): make or use social media icons to replace your link text with images

**Comment by brandaogabriel7** [comment]
## Lesson 11 - CSS Layout
There are some pending issues for this lesson.

- [x] In your index.html file, add a navigational header to your webpage using the `<nav>` element.
- [ ] Add two sections to your index.html file:
    - [x] The first section will be "Skills". Use an h2 tag as you did with your About, Experience, and Connect sections, and be sure to include the id property in the element. You can leave the section empty for now. In later weeks we'll use JavaScript to insert a list of your skills.
    - [ ] The second section will be "Projects". Use an h2 tag as you did with your About, Experience, and Connect sections, and be sure to include the id property in the element. You can leave the section empty for now. In later weeks we'll use JavaScript to pull a list of your project repositories from your GitHub account using their API.
        > You forgot to add the Projects section.
- [x] The navigation menu should include a link to each of your sections (i.e. "About", "Experience", "Skills", "Projects", "Connect")
    - [x] STRETCH GOAL (Optional) Make the header sticky/fixed on the page using code in your index.css file.
- [ ] In your index.css file, change the layout of your "Experience" section using rows and columns so items are displayed in a better layout rather than a list Hint: use Flexbox - we suggest have job titles to the left, dates worked that job to the right and brief description of the job below the title.
> Your experience section content is not what's expected. Instead of the links you're using, you should describe your work experience with job title and date. Then you would use flexbox as described above.
- [ ] Also in your index.css file, update the "Connect" section to use Flexbox to improve the layout of your social media icons or links **NOTE:** This is the basic rubric, but we encourage you to think beyond this list and come up with your own ideas to make your webpage unique!
> You're not using Flexbox in the Connect section..

**Comment by Alarlar** [comment]
> ## Lesson 10 - CSS Basics
> Nice job! Now, the only issues left are:
> 
> 1. Changing the default text color for the page.
> 2. Changing the styles for headings.
> 
> There are comments that explain both of those issues. Below is the updated checklist.
> 
> ### Create and Load Stylesheet
> * [x]  Create a folder called css at the same level as your README.md and index.html files
> * [x]  Inside that folder, create a Cascading Style Sheets (CSS) file called index.css
> * [x]  Open your index.html file
> * [x]  Before the closing  tag, insert a  element with a rel attribute of "stylesheet" and an href attribute that specifies the relative path to your CSS file (i.e. css/index.css)
> 
> ### Write CSS
> * [x]  Change the background color of the page body
> * [ ]  Change the default text color
>   > You only changed the text color for links and nav items. To change the default text color for the whole page, you can set it in the body styles.
> * [x]  Customize the font family
>   
>   * [ ]  STRETCH GOAL (optional): load in a font family from [Google Fonts](https://fonts.google.com/)
> * [x]  Add spacing (padding/margin) between sections
> * [x]  Change the alignment of the content of one of your sections
> * [ ]  Change the font size, weight, and color of headings
> 
> > You did change the heading styles but you used the same styles for all of them and made your name disappear. Take a look at the comment I left about that.
> 
> * [x]  Transform the style of your Name at the top of the page
>   
>   * [ ]  STRETCH GOAL (optional): add a picture of yourself to HTML and/or CSS (remember to include accessibility aspects if you add any images!)
> * [x]  Transform the "Experience" list items into styled blocksto their parent elements (the `#Experience` section). That way you could add many other items without having to repeat this class for all of them.
> * [x]  Transform the style of the "Connect" links
>   
>   * [ ]  STRETCH GOAL (optional): make or use social media icons to replace your link text with images
1. Changing the default text color for the page. - What default color for the page?
2. Changing the styles for headings. - What style should be?

**Comment by Alarlar** [comment]
I think I corrected.

**Pull request merged**